resolver/type_syntax.cc

```diff
@@ -903,13 +903,15 @@ TypeSyntax::ResultType getResultTypeAndBindWithSelfTypeParams(core::MutableConte
 
             auto correctedSingleton = corrected.data(ctx)->singletonClass(ctx);
             auto ctype = core::make_type<core::ClassType>(correctedSingleton);
+            auto originForUninitialized = core::Loc::none();
```

Comment:
ditto comment above; I'd prefer not giving people opportunities to cargo cult `Loc::none` all over the place.